### ISSUE TITLE
Set up github actions for automatic build and deploy

### ISSUE THREAD

**Issue by digama0** [digama0/mmj2#41]
A lot of issues are being caused by the lack of prompt binary releases. It is possible to set up github actions so that it will automatically build mmj2 and create a "release" for the latest version. @david-a-wheeler could I tempt you to try setting this up? My time for mmj2 maintenance is very little these days.

**Comment by david-a-wheeler** [comment]
Yes, I can be tempted :-). I got compilation working with relatively little trouble.

I don't know if GitHub actions are the right way to do this, but I'll check it out.

**Comment by digama0** [comment]
Travis would also work, but I think github actions has more freedom to publish releases. I don't mind either way.

**Comment by david-a-wheeler** [comment]
That makes sense.

This would actually be related to my work to make mmj2 / Metamath much easier to install. The release should be both easy to create AND easy to install.
